src/utils.ts

```diff
@@ -123,6 +123,13 @@ export function extensionConfigurationSet<K extends keyof ExtensionConfiguration
   return getConfiguration().update(key, value, target);
 }
 
+export function shouldModifySetting(key: ModifiableExtension) {
+  const modifySettings = extensionConfiguration(SettingsKey.modifySettings);
+  if (typeof modifySettings == "boolean") return modifySettings;
+  if (modifySettings.includes(key)) return true;
+  return false;
```

Comment:
I guess this could be simplified to `return modifySettings.includes(key);`

Reply:
done